client/src/store/utils/thunkCreators.js

```diff
@@ -93,17 +93,19 @@ const sendMessage = (data, body) => {
 
 // message format to send: {recipientId, text, conversationId}
 // conversationId will be set to null if its a brand new conversation
-export const postMessage = (body) => (dispatch) => {
+export const postMessage = (body) =>async (dispatch) => {
   try {
-    const data = saveMessage(body);
+    const data = await saveMessage(body);
 
     if (!body.conversationId) {
       dispatch(addConversation(body.recipientId, data.message));
+      dispatch(fetchConversations())
     } else {
       dispatch(setNewMessage(data.message));
     }
 
-    sendMessage(data, body);
+    await sendMessage(data, body);
```

Comment:
`sendMessage` is not an async function so no need to add an `await` here.

Reply:
I see TY.